rllib/algorithms/bandit/bandit_tf_model.py

```diff
@@ -1,17 +1,25 @@
 import gymnasium as gym
-import tensorflow_probability as tfp
 
 from ray.rllib.models.modelv2 import ModelV2
 from ray.rllib.models.tf.tf_modelv2 import TFModelV2
 from ray.rllib.utils.annotations import override
 from ray.rllib.utils.framework import try_import_tf
+from ray.rllib.utils.framework import try_import_tfp
 from ray.rllib.utils.typing import TensorType
 
 tf1, tf, tfv = try_import_tf()
+tfp = try_import_tfp()
 
 
 class OnlineLinearRegression(tf.Module if tf else object):
     def __init__(self, feature_dim, alpha=1, lambda_=1):
+        if not tfp:
```

Comment:
Not sure, but to avoid the duplicate error message, maybe we can simply do yet another `try_import_tfp(error=True)` here (besides the one above, which I understand we still need)?